pages/apis/managing_api_tokens.md

```diff
@@ -72,7 +69,7 @@ No, you can change the scope and description of a token, or revoke it, but you c
 
 ### Can I re-add my organization to a token?
 
-Yes, the token owner can re-add the organization to the token from their _API Access Tokens_ settings page.
+If an organization has revoked a token, it cannot be re-added to the token. The token owner would have to create a new token with access to your organization.
```

Comment:
Should we provide a simple "no" upfront?

```suggestion
No. If an organization has revoked a token, it cannot be re-added to the token. The token owner would have to create a new token with access to your organization.
```

Reply:
changed